packages/performance-scripts/runWithJsCpuProfile.ts

```diff
@@ -28,7 +28,7 @@ export async function runWithCpuProfiler<F extends () => any>(
     sampleIntervalMicroSec = +(process.env.PROFILE_SAMPLE_INTERVAL ?? 500), // half a millisecond
   } = {}
 ): Promise<ReturnType<F>> {
-  const maybeNameTimePortion = timestamp ? `_${new Date().toISOString().replace(":","_")}` : "";
+  const maybeNameTimePortion = timestamp ? `_${new Date().toISOString().replace(/:/g, "-").replace(/\./g, "-")}` : "";
```

Comment:
can condense this to one replacement regex:
`/[:.]/g`, which means match `:` or `.`

Reply:
True, updated